### PR TITLE
Feature/24 get random tip query

### PR DESCRIPTION
**Pull request by ShaneUP1**
getRandom tip query created and passing test

### CODE REVIEW

**Review by bryanakitchen — APPROVED** [review]
Nice work!

**Review by RhoneLachner — APPROVED** [review]
Looks Great! Many things to resolve.

File: __tests__/tips.test.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      { tip: 'Walk around the room.' }
+    );
+
+    const res = await request(app)
+      .delete(`/api/v1/tips/${tip.id}`);
+
+    expect(res.body).toEqual(tip);
+  });
+
+  it('returns a random tip via GET', async() => {
```
Comment (franco-ortega) [inline __tests__/tips.test.js:79]: This test looks good.

File: lib/controllers/tips.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  .get('/', (req, res, next) => {
+    Tip
+      .find()
+      .then(tips => res.send(tips))
+      .catch(next);
+  })
+  .get('/random', (req, res, next) => {
+    Tip
+      .getRandom()
+      .then(tips => res.send(tips))
```
Comment (franco-ortega) [inline lib/controllers/tips.js:20]: This looks pretty good. My only question is if this is if the data on line 20 is multiple tips (as indicated above in plural) or a just single tip?

File: lib/models/Tip.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    static async insert({ tip }) {
+      const { rows } = await pool.query(
+        'INSERT INTO tips (tip) VALUES ($1) RETURNING *',
+        [tip]
+      );
+
+      return new Tip(rows[0]);
+    }
+
+    static async getRandom() {
```
Comment (franco-ortega) [inline lib/models/Tip.js:21]: Looks good! Nice to learn that SQL can randomize data.

**Review by franco-ortega — APPROVED** [review]

**Pull request merged**